docs/semgrep-supply-chain/transitive-reachability.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: transitive-reachability
+title: Transitive reachability
+hide_title: true
+description: Learn how transitive reachability identifies vulnerabilities introduced by third-party packages.
+tags:
+ - Semgrep Supply Chain
+---
+
+# Transitive reachability (beta)
+
+Semgrep Supply Chains' transitive reachability analysis helps you identify vulnerabilities introduced by third-party packages and their dependencies. 
+
+
+:::info
+This feature is in private beta. To join, contact [support](/support).
+:::
+
+## Supported languages
+
+Semgrep currently performs transitive reachability analysis for JavaScript projects.
+
+## Definitions
+
+For this article:
+- **First-party code**: first-party code refers to your project, which includes the code that Semgrep scans
+- **Third-party code**: third-party code refers to the dependencies that the first-party code calls or imports
+
+Semgrep uses two overarching categories for project dependencies:
+
+- **Direct**: direct dependencies are those explicitly added by the developer, then called in the first-party code. You can visualize and review direct dependencies using the project's configuration file, such as the lockfile or the manifest file. Semgrep can also generate visual dependency graphs for projects that it scans.
+- **Transitive**: transitive dependencies are those that you include indirectly. For example, this happens when you directly add a dependency, but the dependency that you add then calls another dependency. In other words, a transitive dependency is one that third-party code adds.
```

Comment:
This isn't good parallel structure, FWIW. I would write it as something  like:

direct dependencies are those explicitly added by the/a developer...
transitive dependencies are those not explicitly added by the/a developer but included indirectly...